src/nickel.rs

```diff
@@ -102,7 +102,7 @@ impl Nickel<()> {
 
 impl<D: Sync + Send + 'static> Nickel<D> {
     /// Creates an instance of Nickel with default error handling,
-    /// default Ooptions, and custom data.
+    /// custom options, and custom data.
```

Comment:
Recommend markdowning `Options` if practical, to aid in link following on `docs.rs`.  If too hard, don't.